web_modules/mapbox-gl-geocoder/dist/mapbox-gl-geocoder.css

```diff
@@ -0,0 +1,134 @@
+/* Basics */
```

Comment:
Is this being vendorified? Why does mapbox end up here but not `lodash` and `List.js`?

Reply:
Hmmm, because I forgot to run `npm run snowpack install` after adding lodash... it get's bundled? Sorry! Should be good now.